app/Http/Controllers/UserController.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Enums\Status;
+use App\Http\Requests\Enum;
+use App\Models\User;
+use App\Models\Authentication;
+use App\Http\Requests\UserRequest;
+use Carbon\Carbon;
+use Cookie;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Hash;
+use Illuminate\Support\Facades\Mail;
+use Illuminate\Support\Facades\Validator;
+use Illuminate\Foundation\Auth\RegistersUsers;
+use Illuminate\Auth\Events\Registered;
+use App\Http\Controllers\Controller;
+
+class UserController extends Controller
+{
+    public function create(Request $request)
+    {
+        $enum = $request->enum;
+        $token = $request->token;  
+        $authentication = Authentication::where('token', $token)->orderBy('created_at', 'DESC')->orderBy('id', 'DESC')->first();
+
+        $current_time = now();
+        $token_expiry = $authentication->expired_at;  
+
+        if ($current_time > $token_expiry) {
+            return view('users.token_expired_error');
+        }
```

Comment:
```suggestion
        if ($authentication->expired_at < now()) {
            return view('users.token_expired_error');
        }
```
$current_time、$token_expiryがなくても充分読めるのでこれらは削除してください。

Reply:
承知しました。
削除しました。